react/CozyDialogs/Readme.md

```diff
@@ -15,6 +15,7 @@ Pre-built modals ready to be directly used in applications, based on MUI Dialog.
 * content : `<node>` Content of the modal
 * actions : `<node>` Actions of the modal
 * actionsLayout : `<string>` Can be "row" or "column"
+* ...Mui.ModalProps & ...Mui.DialogProps 
```

Comment:
Can you add a link to the respective components ? I think a phrase would be easier to read.

Reply:
I've done it in https://github.com/cozy/cozy-ui/pull/1640